master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -285,6 +286,18 @@ private[celeborn] class Master(
       executeWithLeaderChecker(null, timeoutDeadApplications())
     case CheckForHDFSExpiredDirsTimeout =>
       executeWithLeaderChecker(null, checkAndCleanExpiredAppDirsOnHDFS())
+    case pb: PbWorkerExclude =>
+      val workersToAdd = new util.ArrayList[WorkerInfo](pb.getWorkersToAddList
+        .asScala.map(PbSerDeUtils.fromPbWorkerInfo).toList.asJava)
+      val workersToRemove = new util.ArrayList[WorkerInfo](pb.getWorkersToRemoveList
+        .asScala.map(PbSerDeUtils.fromPbWorkerInfo).toList.asJava)
+      executeWithLeaderChecker(
+        null,
+        handleWorkerExclude(
```

Comment:
The method `handleWorkerExclude` should not be invoked here. This only processes messages without response.